client/src/index.jsx

```diff
@@ -0,0 +1,12 @@
+var React = require('react');
+var ReactDOM = require('react-dom');
+
+console.log("Server is working")
```

Comment:
consider having this check completed in the server code itself. coupling in this way causes an unnecessary dependency of your server on your client file.